BREAKING.md

```diff
@@ -106,6 +81,28 @@ to be `string | undefined`.
 To improve bundle size, it will be converted to a free function so that this logic is tree-shakeable.
 The corresponding method on `IMergeTreeTexHelper` will also be removed.
 
+### Remove `type` field from `ShareLinkInfoType`
+This field has been deprecated and will be removed in a future breaking change. You should be able to get the kind of sharing link from `shareLinkInfo.createLink.link` property bag.
+
+### Remove `ShareLinkTypes` interface
+`ShareLinkTypes` interface has been deprecated and will be removed in a future breaking change. Singnature of `createOdspCreateContainerRequest` has been updated to now accept `ISharingLinkKind` property instead.
+```diff
+    function createOdspCreateContainerRequest(
+        siteUrl: string,
+        driveId: string,
+        filePath: string,
+        fileName: string,
+-       createShareLinkType?: ShareLinkTypes,
++       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
+    ):
+```
+
+
+### Remove `enableShareLinkWithCreate` from `HostStoragePolicy`
+`enableShareLinkWithCreate` feature gate has been deprecated and will be removed in a future breaking change. If you wish to enable creation of a sharing link along with the creation of Fluid file, you will need to provide `createShareLinkType:ISharingLinkKind` input to the `createOdspCreateContainerRequest` function and enable the feature using `enableSingleRequestForShareLinkWithCreate` in `HostStoragePolicy`
+### Add assertion that prevents sending op while processing another op
```

Comment:
Would this be "actionable" change based on our definition here: https://github.com/microsoft/FluidFramework/wiki/Communicating-breaking-changes

Reply:
I don't think so, I'll remove this entry

Reply:
Actionable change should be from perspective of "current" release. In other words, also, if they are deprecating something in next, but alternative solution is still not available in main I don't think there is point of advertising it.